homeassistant/components/knx/schema.py

```diff
@@ -106,6 +106,7 @@ class BinarySensorSchema:
     DEFAULT_NAME = "KNX Binary Sensor"
 
     SCHEMA = vol.All(
+        # deprecated since 2020.9
```

Comment:
2020.9 has never existed?

Reply:
I meant September 2020. Can change it if you like. 
I added it so we can quickly see if it's there for more than 1-2 years. I think it can safely be removed then.

Reply:
It is confusing with the releases. Please state the correct version it was deprecated in.